README.md

```diff
@@ -1,15 +1,43 @@
-# PANCHANGE - analysis scripts
+# PANCHANGE - factors influencing perceived and objective change in symptoms of anxiety, depression and PTSD during 2020 pandemic in the UK
+
+## Summary
+
 Code repository for Young et al 2020 manuscript "Anxiety, depression and trauma symptom change during the COVID-19 pandemic: Retrospective versus objective assessment"
 
+The project uses data from the [Repeated Assessment of Mental health during a Pandemic (RAMP)](https://rampstudy.co.uk/), COVID-19 Psychiatry and Neurological Genetics (COPING), [Genetic links to Anxiety and Depression](https://gladstudy.org.uk/) (GLAD) and [Eating Disorder Genetics Initiative](https://edgiuk.org/) (EDGI) studies.
+
+***Aims***
+1) Assess current symptoms among individuals with and without a prior diagnosis of an anxious or depressive disorder.  
+2) Assess objective and retrospective symptom change in the GLAD sample.   
+3) If 2. considered sufficiently valid, report on whole RAMP and COPING sample current v retrospective change
+
+***Sensitivity checks / stratifications***
+* perceived change in symptoms
+* Pre-existing mental health diagnosis
+* Chronicity of pre-existing diagnosis 
+* Severity of objective baseline symptoms (GLAD & EDGI)
+* Gender
+* Age
+* Ethnicity (if sufficiently powered to do so)
+* Keyworker / vulnerable individual status
+* COVID-19 status (did they become ill, are they at the time or response)
+
+## Project details
 
+***Authors***   
+Katherine Young, Kirstin purves, Christopher Hübel, Molly Davies, Katie Thompson, Shannon Bristow, Thalia Eley, Gerome Breen
```

Comment:
I imagine this isn't the final authors list, but just in case it was meant to be - we will need to discuss the other COPING investigators as well (e.g. Adrian)

Reply:
Nope, definitely isn't, but we should update when we agree this.